### PR TITLE
Document endpoints regarding project lock and unlock

### PR DESCRIPTION
**Pull request by saraycp**
Document these two endpoints following OpenAPI specifications:

- [x] `/source/{project_name}?cmd=lock`
- [x] `/source/{project_name}?cmd=unlock`

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_cmd_lock.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+post:
+  summary: Locks the project
```
Comment (eduardoj) [inline src/api/public/apidocs-new/paths/source_project_name_cmd_lock.yaml:2]:
```suggestion
  summary: Locks the project.
```
Missing dot at the end of the sentence.

File: src/api/public/apidocs-new/paths/source_project_name_cmd_unlock.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+post:
+  summary: Unlocks the project
```
Comment (eduardoj) [inline src/api/public/apidocs-new/paths/source_project_name_cmd_unlock.yaml:2]:
```suggestion
  summary: Unlocks the project.
```
Missing dot at the end of the sentence.

**Review by eduardoj — APPROVED** [review]

File: src/api/public/apidocs-new/paths/source_project_name_cmd_lock.yaml (hunk 1 above)
Comment (saraycp) [reply src/api/public/apidocs-new/paths/source_project_name_cmd_lock.yaml]: done

File: src/api/public/apidocs-new/paths/source_project_name_cmd_unlock.yaml (hunk 2 above)
Comment (saraycp) [reply src/api/public/apidocs-new/paths/source_project_name_cmd_unlock.yaml]: done

**Review by danidoni — APPROVED** [review]

**Pull request merged**